bundles/org.openhab.ui/web/src/pages/settings/things/inbox/inbox-list.vue

```diff
@@ -34,22 +34,28 @@
                    icon-ios="f7:hand_thumbsup" icon-aurora="f7:hand_thumbsup">
           &nbsp;Approve
         </f7-button>
-        <f7-button @click="performActionOnSelection('copy')" color="blue" class="delete wider-screen display-flex flex-direction-row"
-                   icon-ios="f7:square_on_square" icon-aurora="f7:square_on_square">
-          &nbsp;Copy
-        </f7-button>
-        <f7-button color="blue" class="delete narrower-screen" popover-open=".item-popover">
-          ...
-        </f7-button>
-        <f7-popover class="item-popover" ref="popover" :backdrop="false" :close-by-backdrop-click="true"
-                    :style="{ width: '96px' }" :animate="false">
-          <div class="display-flex justify-content-center" style="width: 100%">
-            <f7-link @click="performActionOnSelection('copy')" color="blue" class="delete display-flex flex-direction-column margin-right"
-                     icon-ios="f7:square_on_square" icon-aurora="f7:square_on_square" popover-close=".item-popover">
-              Copy
-            </f7-link>
-          </div>
-        </f7-popover>
+        <!-- buttons for wider screen -->
+        <template v-if="$f7.width >= 500">
```

Comment:
Is this responsive or do we have to refresh the browser?

Reply:
This is responsive :+1:

Reply:
Cool! TIL